ios/INVRegionMonitor.m

```diff
@@ -138,13 +138,19 @@ - (BOOL)_isRegionMonitoringPossible:(CLAuthorizationStatus)status {
     );
 }
 
-- (void)_sendRegionChangeEventWithIdentifier:(NSString *)identifier didEnter:(BOOL)didEnter didExit:(BOOL)didExit {
+- (void)_sendRegionChangeEventWithIdentifier:(NSString *)identifier didEnter:(BOOL)didEnter didExit:(BOOL)didExit locationManager:(CLLocationManager *)locationManager {
     NSDictionary *body = @{
         @"region": @{
             @"identifier": identifier,
         },
         @"didEnter": @(didEnter),
         @"didExit": @(didExit),
+        @"accuracy": @(locationManager.location != nil ? locationManager.location.horizontalAccuracy : 99999),
+        @"bearing": @(locationManager.location != nil ? locationManager.location.course : 0),
```

Comment:
Yea, I would prefer the iOS naming, but I guess you applied the Android naming. I think most important is that we keep the names the same on both platforms, like you mentioned.

I do wonder about `horizontalAccuracy` vs `accuracy`. On iOS there is also `verticalAccuracy`, so naming it `accuracy` may not be accurate (pun intended). Are there any equivalents in Android, or is there just one accuracy?